go/interpreter/lfvm/converter_test.go

```diff
@@ -88,28 +159,45 @@ func TestConverterCheckInvalidCode(t *testing.T) {
 	}
 }
 
+func TestConverterGenPcMapFailsWithSuperInstructions(t *testing.T) {
+	_, err := GenPcMapWithSuperInstructions([]byte{0x00})
+	if err == nil {
+		t.Errorf("program counter mapping does not support super instructions yet")
+	}
+}
+
 func BenchmarkConvertLongExampleCode(b *testing.B) {
+	converter, err := NewConverter(ConversionConfig{})
+	if err != nil {
+		b.Fatalf("failed to create converter: %v", err)
+	}
 	for i := 0; i < b.N; i++ {
-		clearConversionCache()
```

Comment:
Why no clear between runs? 
I'm not sure what are we benchmarking here. I guess both make sense

Reply:
Since I pass a `nil` value as the hash for the conversion call, the result is anyway not cached.